src/resources/projects/website/listing/item-default.ejs.md

```diff
@@ -31,44 +31,64 @@ return value;
 
 let value = readField(item, field);
 if (value !== undefined) {
-print(`<div class="metadata-value listing-${field}">${listing.utilities.outputLink(item, field, value)}</div>`);  
+print(`<div class="metadata-value listing-${field}">${listing.utilities.outputLink(item, field, value)}</div>`);
  }
 }
 %>
 
-<div class="quarto-post image-<%= imageAlign %>" <%= listing.utilities.metadataAttrs(item) %>>
+::: {.quarto-post .image-<%= imageAlign %> <%= listing.utilities.metadataAttrs(item) %>}
+
 <% if (fields.includes('image')) { %>
-<div class="thumbnail">
-<a href="<%- item.path %>" class="no-external">
+`<div class="thumbnail"><a href="<%- item.path %>" class="no-external">`{=html}
 <% if (item.image) { %>
 <%= listing.utilities.img(itemNumber, item.image, "thumbnail-image", item['image-alt'], item['image-lazy-loading'] ?? listing['image-lazy-loading']) %>
 <% } else { %>
 <%= listing.utilities.imgPlaceholder(listing.id, itemNumber, item.outputHref) %>
 <% } %>
-</a>
-</div>
+`</a></div>`{=html}
 <% } %>
-<div class="body">
+
+::: {.body}
+
 <% if (fields.includes('title')) { %>
-<h3 class="no-anchor listing-title"><a href="<%- item.path %>" class="no-external"><%= item.title %></a></h3>
-<div class="listing-subtitle"><a href="<%- item.path %>" class="no-external"><%= item.subtitle %></a></div>
+`<h3 class="no-anchor listing-title"><a href="<%- item.path %>" class="no-external">`{=html} <%= item.title %> `</a></h3>`{=html}
+<% if (fields.includes('subtitle')) { %>
+`<div class="listing-subtitle"><a href="<%- item.path %>" class="no-external">`{=html} <%= item.subtitle %> `</a></div>`{=html}
+<% } %>
 <% } %>
-<% if (fields.includes('categories') && item.categories) { %> 
-<div class="listing-categories">
+
+<% if (fields.includes('categories') && item.categories) { %>
+`<div class="listing-categories">`{=html}
 <% for (const category of item.categories) { %>
-<div class="listing-category" onclick="window.quartoListingCategory('<%=utils.b64encode(category)%>'); return false;"><%= category %></div>
+`<div class="listing-category" onclick="window.quartoListingCategory('<%- utils.b64encode(category) %>'); return false;">`{=html} <%= category %> `</div>`{=html}
```

Comment:
I don't think `<%=` should be used here, mostly semantic as base64 should not have any markdown syntax in it.

Reply:
We need to do more testing as we might fix the following issue which seems related to how categories, etc. are handled.

- #11752

Reply:
Oh I missed that. I can have a look as I think I fixed something related, after we switch to base64 encoding for category. 

I would probably do a separate PR and not fix everything in one single PR.

Reply:
Yes, if it's in the JS script it makes sense to make a separate PR.